package.json

```diff
@@ -85,6 +85,9 @@
 		]
 	},
 	"dependencies": {
-		"@prisma/client": "^4.12.0"
+		"@prisma/client": "^4.12.0",
+		"azurite": "^3.26.0",
```

Comment:
i think npm has done something funky with the dependencies, as i don't think `azurite` should be a non-dev dependency. I think it's only really needed as a devDependency in `back-office-subscribers`

Reply:
All of this was added from me doing `npm update` and `npm audit` after updating node versions everywhere, not sure why its done but should I manually remove?